compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmTerminatorGen.java

```diff
@@ -1082,21 +1089,33 @@ private void genFPCallIns(BIRTerminator.FPCall fpCall, String moduleClassName, B
             this.mv.visitMethodInsn(INVOKEVIRTUAL, FUNCTION_POINTER, "getFunction", GET_FUNCTION, false);
         }
 
+        boolean workerDerivative = fpCall.workerDerivative;
```

Comment:
Do we need this `boolean` variable creation? We can directly pass the boolean argument.

Reply:
It is used twice, hence used a variable.